scripts/LVGLImage.py

```diff
@@ -1037,10 +1036,10 @@ def pack(r, g, b, a):
                 ):
                     treshold_id = ((y & 7) << 3) + (x & 7)
 
-                    r = (min(r + red_thresh[treshold_id], 0xFF) >> 3) << 3
-                    g = (min(g + green_thresh[treshold_id], 0xFF) >> 2) << 2
-                    b = (min(b + blue_thresh[treshold_id], 0xFF) >> 3) << 3
-                   
+                    r = min(r + red_thresh[treshold_id], 0xFF) & 0xF8
```

Comment:
It seems there's no need to `& 0xF8` etc. manually. The pack method will only keep 5,6,5 bits anyway.

Reply:
It is better to leave it there for purpose of knowing what is actually taking place and that only the higher 5, 6m, and 5 bits of the RGB is what is used.. I do know that the pack if going to bit shift the bottom 3, 2 and 3 bits away anyhow. For the purposes of clarity in the code I do recommend leaving it there. The bit operation is cheap to perform and doesn't cause any kind king of an appreciable difference  in how long it takes to convert the image..

If anything I actually think that perhaps what needs to be done is removing the rgb565 requirement and making it a general flag that will work across all color formats as I am sure it would help with any color anomalies that occur when lowering the color depth in general and not for rgb565 alone.

Reply:
That's fine to me. Thank you.